src/DotNetWorker.Core/Hosting/ServiceCollectionExtensions.cs

```diff
@@ -50,6 +50,9 @@ public static IFunctionsWorkerApplicationBuilder AddFunctionsWorkerCore(this ISe
             // Function Execution Contexts
             services.AddSingleton<IFunctionContextFactory, DefaultFunctionContextFactory>();
 
+            // Function Invocation Manager
+            services.AddSingleton<FunctionInvocationManager>();
```

Comment:
nit: The other singletons (see above in this file) follow a pattern of `services.AddSingleton<Interface, DefaultImplementation>();` 

Is there a reason for the deviation?

Reply:
I didn't create an interface to use for `FunctionInvocationManager`, I can create one for sure but wasn't sure yet if we needed one